src/pages/chat/index.astro

```diff
@@ -1,17 +1,33 @@
 ---
 import { Party } from "../../components/Party";
 import Layout from "../../layouts/Layout.astro";
+import Modal from "../../components/Modal.astro";
 
-const username = Astro.url.searchParams.get("username") ?? "anonymous";
+const usernameParams = Astro.url.searchParams.get("username");
```

Comment:
I updated how we were getting the username and room id here because I was running into issues where the app was working because of missing params. So added an explicit check for null and an empty string and that seemed to work. 

Here is a video of the problem in production

https://github.com/nickytonline/astro-partykit-starter/assets/67210629/de48a4a5-fc8a-459f-a81d-b21f04bdb7fb

Reply:
Mind creating a separate PR for this as it's not directly related to this feature.

I created #36 for this.

Reply:
sure, I can remove the changes from this PR and create a separate one